package.json

```diff
@@ -51,26 +54,29 @@
     "eslint": "^3.2.2",
     "eslint-config-airbnb": "10.0.0",
     "express": "4.14.0",
+    "karma": "^1.3.0",
+    "karma-babel-preprocessor": "^6.0.1",
+    "karma-firefox-launcher": "^1.0.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.1",
+    "karma-webpack": "^1.8.0",
     "minimist": "1.2.0",
+    "mocha": "3.0.2",
+    "mockery": "2.0.0",
     "nock": "8.0.0",
     "nodemon": "1.11.0",
     "npm-run-all": "3.1.1",
     "pug": "2.0.0-beta6",
     "pug-cli": "1.0.0-alpha6",
     "should": "11.1.0",
     "sinon": "1.17.5",
-    "webpack": "1.13.2",
-    "chai": "3.5.0",
-    "mocha": "3.0.2",
-    "jsdom": "9.6.0",
-    "jsdom-global": "2.1.0",
-    "mockery": "2.0.0"
+    "webpack": "1.13.2"
   },
   "scripts": {
     "build": "sh scripts/build.sh",
     "test": "sh scripts/lint.sh && npm-run-all test:*",
     "test:server": "sh scripts/tests.sh",
-    "test:client": "mocha --require babel-register client/test/**/*.js",
+    "test:client": "sh scripts/tests_client.sh",
```

Comment:
Please, don't add new tests in external scripts, but rely on _npm scripts_ and its `pre`/`post` scripts. It do let it be more maintainable / readable (see comments in the script file). Thanks.